src/shared/utils/traverseAllChildren.js

```diff
@@ -188,13 +188,15 @@ function traverseAllChildrenImpl(
           }
         }
       }
+      var childrenString = String(children);
```

Comment:
We use `'' + whatever` in other places. I don't think it really matters since both trigger `whatever.toString()` but might be good to be consistent.